terraform/browser/add_response_security_headers.js

```diff
@@ -8,12 +8,6 @@ function handler(event) {
     // Set HTTP security headers
     // Since JavaScript doesn't allow for hyphens in variable names, we use the
     // dict['key'] notation
-    headers['strict-transport-security'] = {
```

Comment:
This unnecessarily touching an unused function that is slated for removal. See comment at the very top.

Reply:
I've split up my commit in order to separate the intentions. With this PR adding the `Content-Security-Policy` header via the Terraform resource `aws_cloudfront_response_headers_policy`, we also can move the existing security headers out of the JS function and into the same resource.

Since I was able to deploy to `anvildev.broser` with both `add_response….js` files removed, I've added their removal as a first commit in this PR that references #5871 .